amundsen_application/static/js/components/common/ResourceListItem/TableListItem/index.tsx

```diff
@@ -15,38 +13,41 @@ class TableListItem extends React.Component<TableListItemProps, {}> {
     super(props);
   }
 
-
-  /* TODO: We have to fix a bug with this feature. Commented out support.
-  const createLastUpdatedTimestamp = () => {
-    if (lastUpdated) {
-      const dateTokens = new Date(lastUpdated).toDateString().split(' ');
-      return (
-        <label>
-          {`${dateTokens[1].toUpperCase()} ${dateTokens[2]}`}
-        </label>
-      )
-    }
-    return null;
-  }*/
-
   getLink = () => {
-    const { item, logging } = this.props;
-    return `/table_detail/${item.cluster}/${item.database}/${item.schema_name}/${item.name}`
+    const { table, logging } = this.props;
+    return `/table_detail/${table.cluster}/${table.database}/${table.schema_name}/${table.name}`
       + `?index=${logging.index}&source=${logging.source}`;
   };
 
-
   render() {
-    const { item } = this.props;
+    const { table } = this.props;
+
+    // TODO - default last_updated timestamp is just for testing
+    table.last_updated = table.last_updated || Date.now();
+    const dateTokens = new Date(table.last_updated).toDateString().split(' ');
+    const lastUpdated = `${dateTokens[1]} ${dateTokens[2]}, ${dateTokens[3]}`;
+
     return (
-      <li className="list-group-item search-list-item">
-        <Link to={ this.getLink() }>
-          <img className="icon icon-color icon-database" />
-          <div className="resultInfo">
-            <span className="title truncated">{ `${item.schema_name}.${item.name} `}</span>
-            <span className="subtitle truncated">{ item.description }</span>
+      <li className="list-group-item">
+        <Link className="resource-list-item table-list-item" to={ this.getLink() }>
+          <img className="icon icon-database icon-color" />
+          <div className="content">
+            <div className="col-xs-12 col-sm-6">
+              <div className="main-title truncated">{ `${table.schema_name}.${table.name}`}</div>
+              <div className="description truncated">{ table.description }</div>
+            </div>
+            <div className="col-sm-3">
+              <div className="secondary-title">Frequent Users</div>
+              {/*TODO - Replace with a link to a real user*/}
+              <div className="description"><a href="#">Ash Ketchum</a></div>
+            </div>
+            <div className="col-sm-3">
+              <div className="secondary-title">Last Updated</div>
+              <div className="description">
```

Comment:
I also think this column should be smaller and not given the same space as `Frequent Users`. In our designs it seems intentionally further pushed to the right hand side of the list item. 

I think this makes sense because the information in this column is always just going to a be a date that gets formatted in such a way that it's always 11-12 characters and therefore guaranteed to fit into some kind of minimum width. Meanwhile the `Frequent Users` section is an unknown list where we will want to render as many items as possible so it needs more real-estate than an almost constant length date string. Thoughts?

Reply:
Gave the `Frequent Users` column 4 columns when possible, but down to 3 columns when the date needed more space.

Reply:
What content is hidden? I think we should reconsider the idea of hiding any information that Product has deemed useful for our users.

Reply:
`Frequent Users` and `Latest Update` go away in XS screens. There is literally no space to put them in. 

<img width="503" alt="screen shot 2019-03-07 at 3 23 18 pm" src="https://user-images.githubusercontent.com/4338702/53996361-f77af200-40ec-11e9-8c44-c8016459f8fd.png">

We might be able to get away with showing just 2 columns, but at the expense of hiding the table name, which is most important.
<img width="471" alt="screen shot 2019-03-07 at 3 21 13 pm" src="https://user-images.githubusercontent.com/4338702/53996296-b4208380-40ec-11e9-8ca0-d0b20522cd94.png">


That being said, I don't think most people browse Amundsen on mobile so this should be a < 1% issue.

Reply:
We could also put them below, but that would make the search results really long. Probably better to hide them.

Reply:
Is that first image what the current behavior is? If so then I think that's fine. I thought "hidden" meant that the information would just completely go away, but that current view still lets the user know there is information there and that they need to view it on a larger screen.

Reply:
Resolved this conversation offline.